airflow/www/forms.py

```diff
@@ -33,18 +33,20 @@
 from airflow.utils import timezone
 from airflow.www.validators import ValidJson
 
+from .widgets import AirflowDateTimePickerWidget
```

Comment:
```suggestion
from airflow.www.widgets import AirflowDateTimePickerWidget
```

Reply:
I guess. I do wonder why we don't use relative imports more often...

Reply:
I am not sure either. But based on [PEP 8](https://www.python.org/dev/peps/pep-0008/):

>Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages) if the import system is incorrectly configured (such as when a directory inside a package ends up on sys.path):

>However, explicit relative imports are an acceptable alternative to absolute imports, especially when dealing with complex package layouts where using absolute imports would be unnecessarily verbose:

So 🤷‍♂

Reply:
Yeah, the "explicit relative imports" bit is this. In this case it's not overly verbose without it so I guess for consistency i should use absolute.

Reply:
Yup